frontend/public/locale/sv/translation.json

```
@@ -0,0 +1,24 @@
+{
+  "homekey": "Hem",
+  "aboutkey": "Om",
+  "contactkey": "Kontakt",
```

Biome (1.9.4)

[error] The key contactkey was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)